### PR TITLE
Customer opening balance

### PR DESCRIPTION
**Pull request by igorsergiichuk**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #

## The problem

The CustomerOpeningBalance model was absent

## How I resolved it

Add CustomerOpeningBalance model

## How to verify it

Test it!

Notify the following people: @darrynten

### CODE REVIEW

File: tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (darrynten) [inline tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json]: Is this correct? @fergusdixon do you perhaps know?

**Review by darrynten — CHANGES_REQUESTED** [review]

File: tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json (hunk 1 above)
Comment (fergusdixon) [reply tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json]: The response has a body,  in this view the mock looks empty but if I check the files in the branch it has a json response?

Comment (igorsergiichuk) [reply tests/mocks/BankAccountCategory/POST_BankAccountCategory_Save_RESP.json]: I've deleted all the Mocks needed for the BankAccountCategory model. There is another branch for that model.

**Review by darrynten — APPROVED** [review]
Nice

**Pull request merged**